restful_api/myapi/api/views.py

```diff
@@ -36,15 +33,13 @@
 api.add_resource(UserList, "/users", endpoint="users")
 api.add_resource(EmployeeList, "/employees", endpoint="employees")
 api.add_resource(EmployeeResource, "/employees/<int:employee_id>", endpoint="employee_by_id")
-api.add_resource(OfficialEnrollResource, "/official_enroll_by_ids/<int:eng_id>&<int:course_id>", endpoint="official_enroll_by_ids")
-api.add_resource(OfficialEnrollResourceList, "/official_enroll/<int:eng_id>", endpoint="official_enroll")
+api.add_resource(EnrollResource, "/enroll/<int:eng_id>&<int:course_id>&<int:trainer_id>", endpoint="enroll")
```

Comment:
`endpoint="enrollment"` to standardize with the nxt line?

Reply:
Fixed